mpp-ios/ContentView.swift

```diff
@@ -7,16 +7,75 @@
 
 import SwiftUI
 
+struct Station: Identifiable, Hashable {
+    let name: String
+    let id: String
+}
+
 struct ContentView: View {
+    @State private var showingAlert = false
+    
+    var stations = [
+        Station(name: "London Kings Cross", id: "KGX"),
+        Station(name: "Edinburgh Waverley", id: "EDB"),
+        Station(name: "Newcastle", id: "NCL"),
+        Station(name: "York", id: "YRK"),
+        Station(name: "Leeds", id: "LDS")
+    ]
+    @State private var departStation: Station? = nil
+    @State private var arrivalStation: Station? = nil
+    
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundStyle(.tint)
-            Text("Hello, world!")
+            Text("Live Train Times")
+                .font(.title)
+                .fontWeight(.bold)
+                .padding(.all)
+            
+            HStack {
+                Picker("Select departure station", selection: $departStation) {
```

Comment:
To make this easier to read and make the code a bit more DRY, we can bring the picker out in to its own method.

e.g.
```
    private func stationPicker(title: String, selection: Binding<Station?>) -> some View {
        Picker(title, selection: selection) {
            Text("Select...").tag(Optional<Station>(nil))
            ForEach(stations) { station in
                Text(station.name).tag(station as Station?)
            }
        }
        .fixedSize(horizontal: false, vertical: true)
    }
```

which will then turn your Hstack in to something like 
```
          HStack {
              stationPicker(title: "Select departure station", selection: $departStation)
              stationPicker(title: "Select arrival station", selection: $arrivalStation)
          }
```